### PR TITLE
Fix: Show Total Parameters of Model Visualized(#126)

### PR DESCRIPTION
**Pull request by Ram81**
@utsavgarg its the initial work for #126.
abc

### CODE REVIEW

File: ide/views.py [hunk 1]
```diff
@@ -1,5 +1,33 @@
...
+
+@csrf_exempt
+def calculate_parameter(request):
+    if request.method == 'POST':
+        net = yaml.safe_load(request.POST.get('net'))
+        try:
+            netObj = copy.deepcopy(net)
+            for layerId in netObj:
+                for param in netObj[layerId]['params']:
+                    netObj[layerId]['params'][param] = netObj[layerId]['params'][param][0]
```
Comment (utsavgarg) [inline ide/views.py]: Add a comment explaining why is this required

File: ide/views.py [hunk 2]
```diff
@@ -1,5 +1,33 @@
+import sys
```
Comment (utsavgarg) [inline ide/views.py]: Please arrange them in alphabetical order

File: ide/utils/shapes.py [hunk 3]
```diff
@@ -110,6 +110,50 @@ def repeat(layer):
     return shape
 
 
+def handle_concat_layer(outputId, inputId, net):
```
Comment (utsavgarg) [inline ide/utils/shapes.py]: Can't we have a function similar to other layers named `concat` and handle it inside `get_layer_shape`? Do we really need to handle it separately?

File: ide/static/js/content.js [hunk 4]
```diff
@@ -293,6 +369,7 @@ class Content extends React.Component {
       success: function (response) {
         if (response.result === 'success'){
           this.initialiseImportedNet(response.net,response.net_name);
+          this.loadLayerShapes();
```
Comment (utsavgarg) [inline ide/static/js/content.js]: Also, if you are calculating the shapes during import, can you remove the additional calls made to `get_shapes` during export in `export_json`.

Comment (Ram81) [reply ide/static/js/content.js]: Yeah Sure

File: ide/utils/shapes.py (hunk 3 above)
Comment (Ram81) [reply ide/utils/shapes.py]: This is the case for adding the num_output of all the input layers of concat layer, it is needed in topological sort part of shapes.py, where it was being used two times that's why to avoid redundancy I made a separate function

Comment (utsavgarg) [reply ide/utils/shapes.py]: Okay.

**Comment by utsavgarg** [comment]
This calculates the total number of parameters after a model has been imported but does not calculate the change in those parameters after a layer is added or deleted. Parameter calculation has been tested for AlexNet, VGG-16, ResNet-101 and Inception-v3.

**Comment by utsavgarg** [comment]
LGTM, Nice Work :)
